app/controllers/stories_controller.rb

```diff
@@ -75,13 +77,19 @@ def create
     @story = policy_scope(Story).build(allowed_params)
     authorize @story
     @story.requested_by_id = current_user.id unless @story.requested_by_id
-    respond_to do |format|
-      if StoryOperations::Create.call(@story, current_user)
-        format.html { redirect_to project_url(@project) }
-        format.js   { render json: @story }
-      else
-        format.html { render action: 'new' }
-        format.js   { render json: @story, status: :unprocessable_entity }
+    story = BetaStoryOperations::Create.new.call(story: @story, current_user: current_user)
+    Dry::Matcher::ResultMatcher.(story) do |on|
```

Comment:
Since the return of  `call` is not exactly the story, what do you think about renaming this variable to `result`?

```suggestion
    result = BetaStoryOperations::Create.new.call(story: @story, current_user: current_user)
    Dry::Matcher::ResultMatcher.(result) do |on|
```